examples/apache/manifests/init.pp

```diff
@@ -0,0 +1,15 @@
+class { '::apache':
+  default_vhost  => false,
```

Comment:
@sheenaajay the format with `::` is more correct. This avoids issues with finding classes at other parts of the hierarchy.